code/modules/projectiles/gun_attachables.dm

```diff
@@ -723,6 +729,12 @@ inaccurate. Don't worry if force is ever negative, it won't runtime.
 	icon_state = "mosinscope"
 	desc = "A Mosin specific mounted zoom sight scope. Allows zoom by activating the attachment. Use F12 if your HUD doesn't come back."
 
+/obj/item/attachable/scope/standard_magnum
+	name = "R-76 rail scope"
+	desc = "A custom rail mounted zoom sight scope designed specifically for the R-76 Magnum. Allows zoom by activating the attachment. Use F12 if your HUD doesn't come back."
```

Comment:
```suggestion
	desc = "A custom rail mounted zoom sight scope designed specifically for the R-76 Magnum. Allows zoom by activating the attachment."
```

Uh let's not continue this trend. There isn't a case where using a scope should ever make your hud disappear.